rakefile.rb

```diff
@@ -38,32 +38,42 @@
 approval_tests = "tests/FakeItEasy.Tests.Approval/bin/Release/FakeItEasy.Tests.Approval.dll"
 
 repo = 'FakeItEasy/FakeItEasy'
+ci_server_url = 'http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929'
 release_issue_labels = ['P2', 'build', 'documentation']
-release_issue_body = <<-eos
-**Ready** when all other issues on this milestone are **Done** and closed.
+release_issue_common_steps = <<-eos
+**in-progress** when all other issues on this milestone are closed.
 
 - [ ] run code analysis in VS in *Release* mode and address violations (send a regular PR which must be merged before continuing)
-- [ ] if necessary, change `VERSION_SUFFIX` on [CI Server](http://teamcity.codebetter.com/admin/editBuildParams.html?id=buildType:bt929)
+- [ ] if necessary, change `VERSION_SUFFIX` on the [CI Server](#{ci_server_url})
       to appropriate "-beta123" or "" (for non-betas) value and initiate a build
 - [ ] check build
--  edit draft release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases):
+- edit draft release in [GitHub UI](https://github.com/FakeItEasy/FakeItEasy/releases):
     - [ ] complete release notes, mentioning non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
-    - [ ] attach nupkg(s) - main package and/or analyzer, whichever have new content
+    - [ ] attach main nupkg and/or analyzer nupkg, whichever have content to release
     - [ ] publish the release
-- [ ] push NuGet package
+- [ ] push nupkg(s) to NuGet
 - [ ] de-list pre-release or superseded buggy NuGet packages if present
 - [ ] update website with contributors list (if in place)
 - [ ] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
 - [ ] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
-- [ ] post links to the NuGet and GitHub release in each issue in this milestone, with thanks to contributors
-- [ ] run `rake set_version[new_version]` to create a pull request that changes the version in
-       CommonAssemblyInfo.cs to the expected version (of form _xx.yy.zz_)
-- [ ] run `rake create_milestone` (whilst on the branch containing the version update) to:
+- [ ] post links to the GitHub Release in each issue in this milestone, with thanks to contributors
+eos
+
+next_pre_release_steps = <<-eos
+- if there's to be a new pre-release issue
+    - [ ] run `rake pre_release[version_suffix]` to
+        - create a new draft GitHub Release
+        - new issue (like this one) for the next release, adding it to the current milestone
+    - [ ] change `VERSION_SUFFIX` on the [CI Server](#{ci_server_url})
+eos

```

Comment:
I don't think we need to include this in the pre-release issue. We usually don't know if there's going to be another pre-release until we decide to do one, so I think this will usually be redundant noise.